### ISSUE TITLE
Explain, and possibly reduce, arbitrary floor of person risk

### ISSUE THREAD

**Issue by me-and** [microcovid/microcovid#463]
The spreadsheet has a "lowest possible risk" of 120, listed as for the basic and intermediate methods, but seeming to also apply to calculations for the advanced method. I can see the point of having a floor value, but this seems too high to me.

In my situation, following the advanced method, my partner scores very close to zero for their person risk to me, and almost certainly less than 10. Putting that into the spreadsheet calculator, though, I get the floor limit of 120 with the 48% activity risk, for a total score of 58. Given I'm looking at the high-caution budget of 20µCoV/week, it seems simply having a partner puts me at almost three times my budget. That doesn't seem right.

This also doesn't seem to match the behaviour of the website calculator, which appears to not apply this floor value, given the results there for "Uses microCOVID to maintain a risk of 0.1%/year (20 microcovid/week)"

**Comment by blanchardjeremy** [comment]
@me-and Thanks for asking about this. What's the "adjusted prevalence" value in the "locations" sheet your area right now? 

The lowest possible risk in my area is actually 8. 

The lowest possible risk value is calculated as 1/100 of an average person in your area.

I wonder if either A) COVID is blowing up in your area or B) something got messed up on the locations sheet that is showing your prevalence higher than it truly is.

**Comment by me-and** [comment]
Ah, I'd assumed the arbitrary floor was a hardcoded value, rather than calculated, and I didn't think to check!

I'm in England, UK; the estimated true prevalence here according to the spreadsheet / web app is 2.4%. Cases in the UK are … not great at the moment, so I guess that's plausible. My go-to source for UK data is the [ZOE study](https://covid.joinzoe.com/data) which currently predicts 606,715 people have symptomatic Covid in whole of the UK, so a prevalence of 0.89%; I believe ZOE data takes into account underreporting and testing delay in its estimates, but I think it's only counting symptomatic cases, so 2.4% for all cases doesn't seem like an implausible number.

I think the floor is still too high in this circumstance, though. Just thinking about the contact my partner has had with folk other than me, it has all been outside, for at most a couple of minutes, mostly with them at least 6ft if not 10ft away. Even being cautious, I think the spreadsheet is about an order of magnitude too high on that floor.

**Comment by blanchardjeremy** [comment]
Yep. I'd say in that case, just replace the arbitrary floor with "1" or something to remove that value.

I'm having a conversation with the folks on our team who know the science better than I do to see if we want to adjust/remove that feature from the spreadsheet for others as well.

Also, our numbers don't heavily weight asymptomatic cases either, if I'm remembering correctly.  So if your number already accounts for underreporting, you might just use that directly. Or better yet, see if you can find data that are the raw number of reported cases. Then you can let our spreadsheet do its standard underreporting correction.

**Comment by blanchardjeremy** [comment]
I don't know which data sources you find most reliable, but you should be able to get the raw numbers easily here: https://www.nytimes.com/interactive/2020/world/europe/united-kingdom-coronavirus-cases.html

**Comment by beshaya** [comment]
@me-and Thanks for bringing this to our attention; we definitely need to add some explanation for this.

This is a safety brake on the microCovid system, designed to stop the authors from seeing an arbitrarily large number of people who are being arbitrarily safe, and reduce how much close contact happens if local prevalence goes out of control. The arbitrary floor is an agreement among Ibasho that they don't feel comfortable predicting anyone outside their house is more than 100x better than average at avoiding the virus. However, they _do_ allow for their housemates going under the floor.

My suggestion is to manually set your partner's risk to what you think it should be, and perhaps do the same for a small number of people you _really, really_ trust to disclose all their activities and be maximally safe. To hit a target risk of 0.1% chance of infection for a year with a local prevalence of 2.4%, you need to be *1200* times safer than the average person (2.4 / 0.1 * 50). Even going the the grocery store with an n95 mask would use a full week's risk budget.

> Also, our numbers don't heavily weight asymptomatic cases either, if I'm remembering correctly. So if your number already accounts for underreporting, you might just use that directly. Or better yet, see if you can find data that are the raw number of reported cases. Then you can let our spreadsheet do its standard underreporting correction.
@blanchardjeremy our numbers treat asymptomatic the same as symptomatic. Asymptomatic individuals are less effective at transmitting, but not by all that much (about 50% based on https://www.medrxiv.org/content/10.1101/2020.05.10.20097543v2.full.pdf), so you do want to count them.

Looking at ZOE, our numbers are a bit behind - I think if you plugged in the current data you'd get even higher prevalance in the UK. You may, however, want to check your local prevalence. ZOE is predicting about 3x more current infections than tests coming back positive. This seems like a huge underestimate to me, given that England has a 6% positive test rate (which indicates under-testing by at least a factor of 6).

In short, I'm really sorry - it's a dangerous time to be in the UK right now.

Stay safe,
Ben

**Comment by me-and** [comment]
Yeah, having that sort of safety brake makes sense, and I've ended up effectively applying your suggestion of manually setting the risk for my partner, albeit by setting the floor divisor to 500 rather than by setting their risk directly, as I couldn't see a trivial way of doing that.

(I wonder about redesigning the spreadsheet so that the floor is applied on the person risk sheet rather than the activity log sheet; that would make it easier to apply that sort of manual correction to a specific person, as the floor would be being applied once for that person rather than every time they appear in the activity log.)

I think I'm dubious about the factor of 6+, and I note there's [some debate about whether it's too high in the whitepaper](https://www.microcovid.org/paper/7-basic-method#fnref7), too. In particular, while England isn't currently testing everyone who has contact with someone with a positive test, I think we are currently testing a lot of people who are symptomatic. The [methodology ZOE is using to get their prevalence numbers](https://covid.joinzoe.com/post/how-we-are-predicting-covid-in-the-uk) seems reasonable to me, and I think seems more sound than the [sourcing for the multipliers used in the spreadsheet](https://www.microcovid.org/paper/14-research-sources#person-risk). I'm not a statistician, so I don't really trust my own comparison of the two methodologies, but I think I do trust the statisticians at King's College and ZOE to have done a reasonable job with the detailed data available to them.

Which, to be clear, does not mean the spreadsheet and calculator are wrong. They're designed to work with data that's generally available for any given region, while the ZOE data is clearly only available for the UK. I imagine some other regions will have similar datasets to ZOE's, but they're not going to be nearly so directly comparable and available as population, number of tests, and number of positive tests, which is all the current method needs. Plus I expect ZOE's numbers are going for the centre of the confidence interval, whereas the use case for the numbers here means it's much safer to overestimate than underestimate prevalence. However it does seem to be another bit of data pointing at the underreporting factors now being overestimates in at least some cases.

**Comment by beshaya** [comment]
> (I wonder about redesigning the spreadsheet so that the floor is applied on the person risk sheet rather than the activity log sheet; that would make it easier to apply that sort of manual correction to a specific person, as the floor would be being applied once for that person rather than every time they appear in the activity log.)

Yeah, that's a good idea. Internally we have a sheet for the house (basically the people we allow to go below the floor) which handles all the housemate risks. We should probably publish a template for that... @blanchardjeremy?

ZOE's explanation discloses that they are underestimating. Based on my understanding of asymptomatic rates, the true # of cases (asymptomatic + symptomatic, ages 0-100) in the UK may be 2x-7x what they report (# of symptomatic cases age 20-70)

> Does not include asymptomatic COVID infections: there may be a large additional number of these.

In July, about 50% of cases were asymptomatic. A study in October found that areas with universal masking have up to 80% asymptomatic rate. So the true rate in areas with masking may be 5x the symptomatic rate.

> Does not include COVID infections for people aged <20 or >70, since we have too little data logged in the app to model these.
That leaves off about 30% of the population of the UK, so you need to adjust the denominator when calculating the true rate (divide their reported # of infections by the population of 20-70 year olds, not the total population)

It also doesn't seem to take into account the 5 day lag between infection and symptoms, which leads to lagging trends.
ZOE is saying 6M people in the UK are estimated to be symptomatic, or about 1% of the population. Accounting for asymptomatic cases alone, that brings it to 2-5%. Possibly more if we need to correct for people under 20, over 70. That brings the numbers in line with what we compute from test data.

This isn't meant as a criticism of ZOE; they are completely transparent about things they're not taking into account, and don't report a final %.

> In particular, while England isn't currently testing everyone who has contact with someone with a positive test, I think we are currently testing a lot of people who are symptomatic.
They're definitely testing a lot of symptomatic people. But are they testing all? (ZOE says no as well; they estimate 3x more symptomatic cases than positive tests) And that still doesn't address asymptomatic cases.

(Of course, the USA is not doing any better on testing. Sighs...)

**Comment by beshaya** [comment]
Oh, that said, ZOE may have factors leading to over reporting:

> Assumes that app users are representative of the whole population within each segment of location, age-decade and gender. This model does not adjust for other demographics or health information.
Assumes healthy and unhealthy people are equally likely to use our app.
Assumes that app users report the severity of their symptoms in the same way.

It's totally possible that symptomatic people are more likely to use their app than healthy people, so that would lead to a population using their app that is, more likely to be sick than average. I don't think it's possible to know which way that goes without doing a randomized PCR study.

**Comment by blanchardjeremy** [comment]
Hey folks. Curious if we have a path forward on resolving this issue. Are we considering changing the "arbitrary floor"? Or some other path?

**Comment by blanchardjeremy** [comment]
@beshaya regarding adding "pod calculations" to the template, we're working on documentation, which is the first step to getting a template published for pods:

https://docs.google.com/document/d/1TfbcWjxr4kkzGhtGxcX9I4iJuVXrPFitWbTW6iDBzUU/edit

It's currently a very early draft of the documentation. But we are looking for some initial feedback, so feel free to comment.

Hope this helps!

**Comment by blanchardjeremy** [comment]
I'm going to close this as "wontfix" for the time being because I don't see a pressing need. If anyone wants to re-open it in the future, feel free to do so.
